serpentTools/parsers/branching.py

```diff
@@ -121,9 +118,10 @@ def iterBranches(self):
             yield bID, b
 
     def _precheck(self):
-        """todo: is there a good precheck to do here?
+        """Currently, just grabs total number of coeff calcs
         """
-        pass
+        with open(self.filePath) as fObj:
+            self._totalBranches = int(fObj.readline().split()[1])
```

Comment:
Would it be useful to raise errors indicating this may not be a COE file if this fails?

Reply:
Great idea!